packages/web3-validator/src/types.ts

```diff
@@ -143,10 +143,7 @@ export type Schema = {
 };
 export interface Validate {
 	(value: Json): boolean;
-	errors?: ValidationError[];
+	errors?: ZodIssueBase[];
 }
-export type RawValidationError = ValidationError & {
```

Comment:
(breaking change ) it was also exported and will no longer available so : changelog

Reply:
done